cfme/services/requests.py

```diff
@@ -10,16 +10,22 @@
 
 buttons = Region(
     locators=dict(
-        approve="//div[@id='miq_alone' and @title='Approve this Request']/img",
-        deny="//div[@id='miq_alone' and @title='Deny this Request']/img",
-        copy="//div[@id='miq_alone' and @title='Copy original Request']/img",
-        edit="//div[@id='miq_alone' and @title='Edit the original Request']/img",
-        delete="//div[@id='miq_alone' and @title='Delete this Request']/img",
+        approve={version.LOWEST: "//div[@id='miq_alone' and @title='Approve this Request']/img",
+            "5.5": "//button[@name='miq_request_approve' and @title='Approve this Request']/img"},
+        deny={version.LOWEST: "//div[@id='miq_alone' and @title='Deny this Request']/img",
+            "5.5": "//button[@name='miq_request_deny' and @title='Deny this Request']/img"},
+        copy={version.LOWEST: "//div[@id='miq_alone' and @title='Copy original Request']/img",
+            "5.5": "//button[@name='miq_request_copy' and @title='Copy original Request']/img"},
+        edit={version.LOWEST: "//div[@id='miq_alone' and @title='Edit the original Request']/img",
+            "5.5": "//button[@name='miq_request_edit' and @title='Edit the original Request']/img"},
+        delete={version.LOWEST: "//div[@id='miq_alone' and @title='Delete this Request']/img",
+            "5.5": "//button[@name='miq_request_delete' and @title='Delete this Request']/img"},
```

Comment:
I wonder if we could do just like this: `//*[@title='...']/img`, if it is unique, then no verpicking would be required.